product_docs/docs/bdr/4/release-notes.mdx

```diff
@@ -1,9 +1,118 @@
+    `bigserial` and Galloc sequences for `serial` column type.
+
+-   PGD CLI (BDR-1983)
+
+    This BDR release includes a new command-line utility called `pgd` that
+    allows users to do various cluster state checks and to fetch information
+    about cluster.
+
+    This will in future releases become the primary CLI tool for both BDR and HARP.
+
+-   Simplified synchronous replication configuration (BDR-1828)
+
+    New syntax for specifying the synchronous replication options, with focus
+    on BDR groups and SQL based management (as opposed to config file).
+
+    In future versions this will also replace the current Eager Replication and
+    CAMO configuration options.
+
+-   Group Commit (BDR-1828)
+
+    The initial kind of synchronous commit that can be configured via the new
+    configuration syntax.
+
+-   Allow a Raft request to be required for CAMO switching to Local Mode (BDR-1925, RT78928)
+
+    Add a `require_raft` flag to the CAMO pairing configuration which
+    controls the behavior of switching from CAMO protected to Local
+    Mode, introducing the option to require a majority of nodes to be
+    connected to allow to switch to Local Mode.
+
+-   Allow replication to continue on `ALTER TABLE ... DETACH PARTITION CONCURRENTLY`
+    of already detached partition (RT78362, BDR-1958)
+
+    Similarly to how BDR 4 handles `CREATE INDEX CONCURRENTLY` when same index
+    already exists, we now allow replication to continue when
+    `ALTER TABLE ... DETACH PARTITION CONCURRENTLY` is receiver for partition
+    that has been already detached.
+
+-   Implement buffered write for LCR segment file (BDR-1421)
+
+    This should reduce I/O and improve CPU usage of the Decoding Worker.
+
+-   Add `fast` argument to bdr.alter_subscription_disable() (BDR-2017, RT79798)
+
+    The argument only influences the behavior of `immediate`.
+    When set to `true` (default) it will stop the workers without letting them
+    finish the current work.
+
+-   Simplify `bdr.{add,remove}_camo_pair` functions to return void (BDR-1961)
+
+### Resolved Issues
+
+-   Keep the `lock_timeout` as configured on non-CAMO-partner BDR nodes (BDR-1916)
+
+    A CAMO partner uses a low `lock_timeout` when applying transactions
+    from its origin node.  This was inadvertently done for all BDR nodes
+    rather than just the CAMO partner, which may have led to spurious
+    `lock_timeout` errors on pglogical writer processes on normal BDR
+    nodes.
+
+-   Show a proper wait event for CAMO / Eager confirmation waits (BDR-1899,RT75900)
+
+    Show correct "BDR Prepare Phase"/"BDR Commit Phase" in `bdr.stat_activity`
+    instead of the default “unknown wait event”.
+
+-   Reduce log for bdr.run_on_nodes (BDR-2153, RT80973)
+
+    Don't log when setting `bdr.ddl_replication` to off if it's
+    done with the "run_on_nodes" variants of function. This eliminates
+    the flood of logs for monitoring functions.
+
```

Comment:
Do we want to mention about BDR-2038 / RT80204 over here ?

Reply:
This is not relevant to RT80204